examples/angular/src/app/pages/services/page.service.ts

```diff
@@ -45,54 +39,38 @@ export class PageService {
      * @return {*}  {(Observable<{ page: DotCMSPageAsset | { error: PageError }; nav: DotcmsNavigationItem }>)}
      * @memberof PageService
      */
-    getPageAndNavigation(route: ActivatedRoute, config: any): Observable<PageAndNavResponse> {
-        if (!this.navObservable) {
-            this.navObservable = this.fetchNavigation(route);
-        }
-
-        return forkJoin({
-            nav: this.navObservable,
-            pageAsset: this.fetchPage(route, config)
-        }).pipe(
-            map(({ nav, pageAsset }) => {
-                const { page, error } = pageAsset;
-
-                return { nav, page, error };
+    getPageAsset<TPage extends DotCMSPageAsset, TContent>(
+        route: ActivatedRoute,
+        extraQueries?: DotCMSPageRequestParams['graphql']
```

Comment:
Why not have this receive the path and pageParams, just like the original method?
It might help keep the usage consistent and more flexible